### PR TITLE
Document op coverage

### PR DESCRIPTION
**Pull request by marbre**

### CODE REVIEW

File: docs/scf-op-coverage.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# SCF Op Coverage
+
+The table below shows the supported SCF ops.
+
+| op                    | supported          | comment |
+| :-------------------- |:------------------:| :------ |
+| for                   | :heavy_check_mark: | |
+| if                    | :heavy_check_mark: | |
+| yield                 | :heavy_check_mark: | |
```
Comment (OliverScherf) [inline docs/scf-op-coverage.md:9]:
```suggestion
| yield                 | :heavy_check_mark: | |

```

File: docs/std-op-coverage.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+| op                    | supported          | comment |
+| :-------------------- |:------------------:| :------ |
+| br                    | :white_check_mark: | via `emitc-translate` |
+| call                  | :white_check_mark: | via `emitc-translate` |
+| cond_br               | :white_check_mark: | via `emitc-translate` |
+| constant              | :white_check_mark: | via `emitc-translate` |
+| index_cast            | :heavy_check_mark: | |
+| return                | :white_check_mark: | via `emitc-translate` |
+| splat                 | :heavy_check_mark: | |
```
Comment (OliverScherf) [inline docs/std-op-coverage.md:13]:
```suggestion
| splat                 | :heavy_check_mark: | |

```

File: docs/tensor-op-coverage.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Tensor Op Coverage
+
+The table below shows the supported Tensor ops.
+
+| op                    | supported          | comment |
+| :-------------------- |:------------------:| :------ |
+| extract               | :heavy_check_mark: | |
```
Comment (OliverScherf) [inline docs/tensor-op-coverage.md:7]:
```suggestion
| extract               | :heavy_check_mark: | |

```

**Review by OliverScherf — CHANGES_REQUESTED** [review]
LGTM, I just added an empty line in every file.

**Comment by marbre** [comment]
> LGTM, I just added an empty line in every file.

Those file already have a newline at the end of every file. Right now this is consistent with all the other op coverage docs we have.

**Comment by OliverScherf** [comment]
> > 
> 
> Those file already have a newline at the end of every file. Right now this is consistent with all the other op coverage docs we have.

Okay, they are not shown in the review interface. But fine if you checked.

**Review by OliverScherf — APPROVED** [review]

**Pull request merged**